pkg/apis/eventing/v1beta2/eventtype_defaults_test.go

```diff
@@ -37,13 +37,7 @@ func TestEventTypeDefaults(t *testing.T) {
 		"nil spec": {
 			initial: EventType{},
 			expected: EventType{
-				Spec: EventTypeSpec{
-					Reference: &duckv1.KReference{
-						APIVersion: "eventing.knative.dev/v1",
-						Kind:       "Broker",
-						Name:       "default",
-					},
-				},
+				Spec: EventTypeSpec{},
```

Comment:
Is the new default behaviour to have `EventType.Spec.Reference = nil`?

Reply:
the `default` broker was lame